lms/djangoapps/grades/config/waffle.py

```diff
@@ -10,13 +10,73 @@
+# .. toggle_creation_date: 2017-04-11
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/14771
+# .. toggle_status: supported
+# .. toggle_warnings: This requires the PersistentGradesEnabledFlag to be enabled.
 ASSUME_ZERO_GRADE_IF_ABSENT = u'assume_zero_grade_if_absent'
+# .. toggle_name: grades.disable_regrade_on_policy_change
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: When enabled, a change in grading policy will not trigger re-grading.
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2017-08-03
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/15733
+# .. toggle_status: supported
 DISABLE_REGRADE_ON_POLICY_CHANGE = u'disable_regrade_on_policy_change'
 
 # Course Flags
+
+# .. toggle_name: grades.rejected_exam_overrides_grade
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: When enabled, a rejected exam will trigger a grade override. Note that this flag is not used
+#   in edx-platform, but only in edx-proctoring.
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2019-05-29
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/20719
+# .. toggle_status: supported
 REJECTED_EXAM_OVERRIDES_GRADE = u'rejected_exam_overrides_grade'
+# .. toggle_name: grades.rejected_exam_overrides_grade
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: When enabled, grades can no longer be updated 30 days after a course has ended. Note that this
+#   is only valid for courses which actually have an end date.
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2018-10-01
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/19026
+# .. toggle_status: supported
 ENFORCE_FREEZE_GRADE_AFTER_COURSE_END = u'enforce_freeze_grade_after_course_end'
+
+# .. toggle_name: grades.writable_gradebook
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: When enabled, add GET/POST endpoints for updating gradebook entries in bulk. Also, a link to
+#   the writable gradebook is added to the instructor dashboard.
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2018-10-03
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/19054
+# .. toggle_warnings: Enabling this requires that the `WRITABLE_GRADEBOOK_URL` setting be properly defined.
+# .. toggle_status: supported
 WRITABLE_GRADEBOOK = u'writable_gradebook'
+
+# .. toggle_name: grades.bulk_management
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: When enabled, bulk features are visible for management in masters course. As far
+#   as we understand, this feature is now unused and obsolete.
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2019-08-20
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/21389
+# .. toggle_status: supported
```

Comment:
At some point, we'll want to review all of these grade-related toggles.  Grades is a complex part of the platform.  Investing in removing any and all of these toggles will be valuable for the long-term.  I don't believe many of these need to be `open_edx` use-cases.

Perhaps we need to move forward with these docs for now - and then do a 2nd pass later to revisit all the toggles marked as `open_edx`.  Thoughts?

Reply:
I’d prefer a ‘?’ or blank for things we aren’t sure about and want to revisit. And, I agree that we don’t need the documentation to be perfect before landing this.

Reply:
I'll add this to my list of open questions in this PR's description and keep this as it is for now.